CHANGELOG.md

```diff
@@ -7,8 +7,8 @@ and this project, at least loosely, adheres to [Semantic Versioning](https://sem
 ## Unreleased
 ### Changed
 ### Added
-- method to identify mask parameters with no TOAs and optionally freeze them
```

Comment:
Why is this line deleted?

Reply:
I don't know if that was intentional - changelog merges may have had issues